SPECS/pixman/pixman.spec

```diff
@@ -1,10 +1,11 @@
 Summary:	pixel manipulation library.
 Name:		pixman
 Version:    0.36.0
-Release:    2%{?dist}
+Release:    3%{?dist}
 License:	MIT
 URL:        https://cgit.freedesktop.org/pixman/
 Source0:    https://xorg.freedesktop.org/archive/individual/lib/%{name}-%{version}.tar.bz2
+Patch0:     CVE-2022-44638.patch
```

Comment:
Nit: spacing

Reply:
spacing?

Reply:
The Summary, Name, version, release and urls usually all line up nicely. Not a change you added but just looks weird as is. Feel free to ignore if needed.